### ISSUE TITLE
Inconsistent braces in example on the website for metadata filtering

### ISSUE THREAD

**Issue by c4ndyfl1p** [deepset-ai/haystack-website#390]
The last example for nested filters on the website at : [https://haystack.deepset.ai/guides/metadata-filtering](https://haystack.deepset.ai/guides/metadata-filtering) doesn't run due to inconsistent curly braces.

Specifically this bit:
```{"$or": [
    {"$and": {
        "Type": "News Paper",
        "Date": {"$lt": "2019-01-01"}
        }
    },
    "$and": {
        "Type": "Blog post",
        "Date": {"$gte": "2019-01-01"}
        }
    }]
}
```

**Comment by c4ndyfl1p** [comment]
Fixed it in #388

**Comment by brandenchan** [comment]
Thank you so much for picking this up!
